### PR TITLE
Update statistics related things to use EBNF etc

### LINKED ISSUE #17463: statistics.md

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Closes #17463 

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by dveeden** [comment]
@terry1purcell could you review this?

**Review by kennytm — APPROVED** [review]

**Comment by qiancai** [comment]
@dveeden, for sql-statements/sql-statement-drop-stats.md, I've restructured the content in [`8fa7e07` (#17464)](https://github.com/pingcap/docs/pull/17464/commits/8fa7e07bea228bf90fedff489d47631c8921c862) to add the basic usage info of `DROP STATS` to the `## Usage` section. PTAL. Thanks.

**Comment by qiancai** [comment]
there is still one comment in https://github.com/pingcap/docs/pull/17464/files#r1602703834, which is not shown by default because large diffs in `statistics.md` are not rendered by default.

**Comment by dveeden** [comment]
> There is still one comment in https://github.com/pingcap/docs/pull/17464/files#r1602703834, which is not shown by default because large diffs in `statistics.md` are not rendered by default.

Yes, I'll have a look at that.

File: statistics.md [hunk 1]
```diff
@@ -459,30 +432,7 @@ You can view the `ANALYZE` status and statistics information using the following
...
-| table_name | The table name |
-| partition_name| The partition name |
-| job_info | The task information. If an index is analyzed, this information will include the index name. When `tidb_analyze_version =2`, this information will include configuration items such as sample rate. |
-| processed_rows | The number of rows that have been analyzed |
-| start_time | The time at which the task starts |
-| state | The state of a task, including `pending`, `running`, `finished`, and `failed` |
-| fail_reason | The reason why the task fails. If the execution is successful, the value is `NULL`. |
-| instance | The TiDB instance that executes the task |
-| process_id | The process ID that executes the task |
+When executing the `ANALYZE` statement, you can view the current state of `ANALYZE` using [`SHOW ANALYZE STATUS`](/sql-statements/sql-statement-show-analyze-status.md).
```
[the remark this answers is not in the document]
Comment (dveeden) [reply statistics.md:435]: Done in d07d2a81fe91813b7778cae03f9e26977521a82a

**Comment by dveeden** [comment]
@qiancai PTAL

**Review by qiancai — APPROVED** [review]

**Pull request merged**